asynctest/templates.nim

```diff
@@ -1,6 +1,18 @@
 template suite*(name, body) =
+
   suite name:
 
+    # Runs before all tests in the suite
+    template before(beforeBody) {.dirty, used.} =
+      let b = proc {.async.} = beforeBody
+      waitFor b()
+
+    # Runs after all tests in the suite
+    template after(afterBody) {.dirty, used.} =
+      template suiteAfterIMPL: untyped {.dirty.} =
```

Comment:
Similar comment w.r.t. naming.

I think you can get rid of the dirty pragma here as well, if you use an inject pragma. I think it should work, but I haven't tested it.

```suggestion
    template teardownSuite(afterBody) {.used.} =
      template suiteAfterIMPL: untyped {.inject.} =
```